src/method-management/method-management.ts

```diff
@@ -0,0 +1,23 @@
+import { components } from "./models/schema";
+
+type MfaMethod = components["schemas"]["MfaMethod"];
+
+export interface Response {
+  statusCode: number;
+  body: string;
+}
+
+export const handler = async (): Promise<Response> => {
+  const response: MfaMethod = {
```

Comment:
Should this response be an array of `MfaMethod`s?

Reply:
Yes, it should.  Updated.